test/integration/concrete/lockup-linear/create-with-durations/createWithDurations.t.sol

```diff
@@ -28,30 +28,6 @@ contract CreateWithDurations_LockupLinear_Integration_Concrete_Test is
         expectRevertDueToDelegateCall(success, returnData);
     }
 
-    function test_RevertWhen_CliffDurationCalculationOverflows() external whenNotDelegateCalled {
```

Comment:
In the original test, it is true that `cliffDuration` was set so that `cliffTime` would wind up having a value of 0. But that was by choice. We could have made `cliffDuration` have a value of 1 by adding `2 seconds` instead of `1 seconds`.

What am I missing? It seems to me that this test should be brought back.